src/main/kotlin/no/nav/familie/ba/sak/behandling/vedtak/vedtaksperiode/VedtaksperiodeMedBegrunnelserController.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.ba.sak.behandling.vedtak.vedtaksperiode
+
+import no.nav.familie.ba.sak.behandling.fagsak.FagsakService
+import no.nav.familie.ba.sak.behandling.restDomene.RestFagsak
+import no.nav.familie.ba.sak.behandling.restDomene.RestPutVedtaksperiodeMedBegrunnelse
+import no.nav.familie.ba.sak.behandling.steg.BehandlerRolle
+import no.nav.familie.ba.sak.sikkerhet.TilgangService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.ResponseEntity
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/vedtaksperiode")
```

Comment:
```suggestion
@RequestMapping("/api/vedtaksperioder")
```
https://restfulapi.net/resource-naming/